certified-connectors/CB Blockchain Seal/Readme.md

```diff
@@ -0,0 +1,21 @@
+## CB Blockchain Seal
```

Comment:
Can you please update readme.md file with required fields, kindly refer the below link.

Reply:
Hello @vmanoharas ,

I've changed the title of the Readme.md file from "CB Blockchain Seal" to "CB Blockchain Seal Connector" and fixed misstyped operations. I believe the rest is fine.

Thank you,